src/repos/NotificationRepo.ts

```diff
@@ -136,6 +136,46 @@ const notifyNewFlyers = async (flyerIDs: string[]): Promise<void> => {
     });
   });
 };
+/**
+ * Send notifications for new flyers, edited flyers, or deleted flyers by organizations followed by the user.
+ * @param flyerID ID of the flyer being added/changed/deleted
+ * @param bodyText a string containing the body of the notification
+ * @param action a string indicating the action being peformed, a for add, e for edit, d for delete
+ */
+const notifyFlyersForOrganizations = async (
+  flyerID: string,
+  bodyText: string,
+  action: Actions,
+): Promise<void> => {
+  const flyer = await FlyerRepo.getFlyerByID(flyerID); // eslint-disable-line
+  const organization = await OrganizationRepo.getOrganizationBySlug(flyer.organizationSlug);
+  const followers = await UserRepo.getUsersFollowingOrganization(flyer.organizationSlug);
+  let notifTitle = '';
+  let notifBody = '';
+  followers.forEach(async (follower) => {
+    if (action === Actions.ADD) {
+      notifTitle = `New ${organization.name} Event!`;
+      // the format for toDateString is Day of the Week Month Date Year ex: Tue Sep 05 2023
+      notifBody = `${flyer.title} on ${flyer.startDate.toDateString()} at ${flyer.location}`;
+    }
+    if (action === Actions.EDIT) {
+      notifTitle = `New Update from ${organization.name}!`;
+      notifBody = `${flyer.title} has ${bodyText}`;
+    }
+    if (action === Actions.DELETE) {
+      notifTitle = `${organization.name} Event Deleted`;
+      // the format for toDateString is Day of the Week Month Date Year ex: Tue Sep 05 2023
+      notifBody = `${flyer.title} on ${flyer.startDate.toDateString()} has been removed`;
+    }
```

Comment:
Can we use a switch statement instead? Not sure if the below code works but give it a try.

```suggestion
    switch (action) {
      case Action.Add:
        notifTitle = `New ${organization.name} Event!`;
        // the format for toDateString is Day of the Week Month Date Year ex: Tue Sep 05 2023
        notifBody = `${flyer.title} on ${flyer.startDate.toDateString()} at ${flyer.location}`;
      case Action.Edit:
        notifTitle = `New Update from ${organization.name}!`;
        notifBody = `${flyer.title} has ${bodyText}`;
      case Action.Delete:
        notifTitle = `${organization.name} Event Deleted`;
        // the format for toDateString is Day of the Week Month Date Year ex: Tue Sep 05 2023
        notifBody = `${flyer.title} on ${flyer.startDate.toDateString()} has been removed`;
    }
```